src/components/BookSearch/BookItem.js

```diff
@@ -21,20 +19,7 @@ export default function BookItem({ book, margin, radius }) {
       >
         <Row>
           <Col md="auto" xs="auto" className={styles.bookImgWrapperCol}>
-            <img
-              className={`${styles.cardImg} ${imgObjectFitStyle}`}
-              src={book.thumbnail}
-              onLoad={(img) => {
-                if (img.currentTarget.clientWidth == 180) {
```

Comment:
Podczas implementacji znika ten if, nie pamiętam dokładnie co on robił ale jeżeli tu jest to był potrzebny, przywróć tą mechanike do ImageWithShimmer.js